dropshots/src/main/java/com/dropbox/dropshots/Dropshots.kt

```diff
@@ -285,23 +382,13 @@ internal fun isRecordingScreenshots(rootScreenshotDirectory: File): Boolean {
 }
 
 /**
- * Creates a filename from the requested name that'll be 64 characters or less.
- *
- * If the requested name is longer that 64 characters, it'll be shortened by
- * encoding the end of the name, leaving the beginning in tact to hopefully provide
- * somewhat human readable names while still trying to prevent collisions.
+ * Creates a filename from class-name and test-name of where the screenshot was taken.
  */
-internal val defaultFilenameFunc = { testName: String ->
-  if (testName.length < 64) {
```

Comment:
This exists to meet a requirement in Changes. We don't **need** this in the public Dropshots library, but without it we might need to override the default in our internal uses.

Reply:
Reverted!